service-workers/service-worker/resources/import-scripts-diff-resource-map-worker.js

```diff
@@ -0,0 +1,10 @@
+importScripts('/resources/testharness.js');
+
+let echo1 = null;
+    echo2 = null;
```

Comment:
Add `let`, or use commas rather than semicolons.

Reply:
Updated in latest commit.